### PR TITLE
added new cid

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: cid-redirects.json [hunk 1]
```diff
@@ -1619,6 +1619,9 @@
   "/cid/1989": "/docs/observability/aws/integrations/aws-api-gateway",
   "/cid/1990": "/docs/observability/aws/integrations/aws-application-load-balancer",
   "/cid/1991": "/docs/integrations/amazon-aws/lambda",
+  "/cid/2003": "/docs/integrations/amazon-aws/api-gateway/",
```
Comment (JainM6) [inline cid-redirects.json:1622]:
```suggestion
  "/cid/2003": "/docs/integrations/amazon-aws/api-gateway",
```

File: cid-redirects.json [hunk 2]
```diff
@@ -1619,6 +1619,9 @@
...
+  "/cid/2007": "/docs/integrations/amazon-aws/elastic-container-service",
+  "/cid/2034": "/docs/integrations/amazon-aws/elastic-container-service-container-insights-cloudwatch",
```
Comment (JainM6) [inline cid-redirects.json:1624]: Can you check the name again? Do we have this doc this place?

**Review by JainM6 — CHANGES_REQUESTED** [review]
Minor fix and one query.

File: cid-redirects.json (hunk 2 above)
Comment (JV0812) [reply cid-redirects.json:1624]: I have checked and found cid link only for aws lambda and there were no cid number for the other paths.

Comment (JainM6) [reply cid-redirects.json:1624]: @JV0812 Thank you.

**Review by JainM6 — APPROVED** [review]

**Pull request merged**